duckdns/data/run.sh

```diff
@@ -53,19 +53,31 @@ fi
 # Run duckdns
 while true; do
 
-    [[ ${IPV4} != *:/* ]] && ipv4=${IPV4} || ipv4=$(curl -s -m 10 "${IPV4}")
-    [[ ${IPV6} != *:/* ]] && ipv6=${IPV6} || ipv6=$(curl -s -m 10 "${IPV6}")
+    # ipv4
+    if [[ ${IPV4} == "none" ]]; then # if none, don't update ipv4 entry
+        ipv4=""
+    elif [[ ${IPV4} != *:/* ]]; then # if not a URL, use as-is
+        ipv4="&ip=${IPV4}"
+    else
+        ipv4="&ip=$(curl -s -m 10 "${IPV4}")"
+    fi
+    # ipv6
+    if [[ ${IPV6} == *:*:* ]]; then
+        ipv6="&ipv6=${IPV6}"
+    else
+        ipv6="&ipv6=$(bashio::network.ipv6_address "${IPV6}" | head -n1 | cut -d/ -f1)" || ipv6=""
```

Comment:
It is usual in IPv6 networks to assign multiple addresses to an adapter,  Does this method always return a global scope address that can be used to reach the device?